starknet_devnet/blocks.py

```diff
@@ -152,3 +152,50 @@ async def generate(
         self.__state_updates[block_number] = state_update
 
         return block
+
+    def generate_empty(
```

Comment:
I see a lot of overlap with the `generate` method. In a future PR, we could have `generate_empty` as a special case of `generate`, I have some ideas how it could be done.